### DISCUSSION TITLE
LDAP integration

### DISCUSSION THREAD

**Discussion by timothysing-anzo** [drakkan/sftpgo#793] [Q&A]
Complete and utter newb here. 

Im really struggling to understand how to use the ldap examples to allow SFTPGO to use ldap for user auth..

Do you have any tutorials which shows how to use these modules with an already running docker instance of sftpgo?

tx

**Comment by drakkan** [comment]
Hi,

there is no step by step tutorial, sorry. For the script based hook I think it is enough to make the executable available to the running instance, for example you can copy it to a volume and reference by path. For the HTTP based hook you need to run it somewhere (for example another docker instance or a dedicated host/VM).

Please note that LDAP hooks are just examples to use as a starting point, you will probably need to adapt them to your use case
